### DISCUSSION TITLE
ANARI for Visualization

### DISCUSSION THREAD

**Discussion by bigdimboom** [isl-org/Open3D#6328] [Ideas]
The KHRONOS Group has released the standardized specifications for scientific rendering and visualization engines named ANARI. We might think about incorporating ANARI into open3d. One of the benefits is that users will have a variety of back-end implementations to select from.

see: https://www.khronos.org/api/index_2017/anari

**Comment by ssheorey** [comment]
Hi @bigdimboom thanks for bringing this to the Open3D community. This looks like a promising direction to explore in the future. Our current rendering engine - [filament](https://github.com/google/filament) is cross-platform and supports all of Open3D's requirements, but is not ANARI compatible yet. As such, we would need a compelling use case enabled by ANARI to switch the rendering engine backend.

**Comment by germanros1987** [comment]
@bigdimboom are you interested in working on this feature? I think it would be a great addition to Open3D.
